# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libv4l/no-contrib-test.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

SDL2 is not required by the project but it is for the contrib/test subdir so
don't build it until it becomes optional or is fixed upstream as it breaks
cross-build targets when sdl2 is not in pkgsel.

  - NoTag <notag@t2sde.org>

--- v4l-utils-1.26.1/contrib/meson.build.vanilla	2024-01-16 16:08:35.142279049 +0100
+++ v4l-utils-1.26.1/contrib/meson.build	2024-01-16 16:08:38.778169896 +0100
@@ -1,5 +1,4 @@
 if host_machine.system() == 'linux'
-    subdir('test')
     subdir('xc3028-firmware')
     if have_i2c_dev
         subdir('rds-saa6588')
